### PR TITLE
doc: add MAINTAINERS

### PR DESCRIPTION
**Pull request by fanminshi**

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Fanmin Shi <fanmin.shi@coreos.com> (@fanminshi) pkg:*
+Luca Burgazzoli <lburgazzoli@gmail.com> (@lburgazzoli) pkg:*
```
Comment (xiang90) [inline MAINTAINERS]: new line please.

**Comment by xiang90** [comment]
lgtm

**Comment by xiang90** [comment]
probably add me and @heyitsanthony to the list initially... so we can help to approve prs.

eventually we might be revoked though :P

**Comment by fanminshi** [comment]
done!

**Pull request merged**